project/project.godot

```diff
@@ -8,6 +8,28 @@
 
 config_version=4
 
```

Comment:
Does Mob and MobSpawner actually need to be in the global scope?

Reply:
Ah, interesting. I am not sure. I think this is because I made them their own, named classes for re-use. If the ability to reuse them is useful, then maybe they need to be global. Maybe they are not required. Here's an image of what it enables:

![image](https://user-images.githubusercontent.com/23508546/220417295-e1c36ed5-b750-48c1-b22e-dded8a4299e1.png)

I am trying to work out the tangible utility. It might be worth a discussion to keep trying it out or not. @loteque what do you think?

Reply:
Is it true that we only need one mob spawner in the game? If that is the case Im not sure where we would reuse it? Do you have a reuse case?

Reply:
For different enemy waves, you think we should just update the spawning setting for the spawner? I was thinking it would be nice to be able to visualize in the node tree a sequence of waves of different enemies

* MobWaveSequence
    * MobWave
        * BiterPlant
    * MobWave
        * VinePlant
    
Can you chat about how we handle reusability? I am trying to sort out the best way to do this overall and there are several methods.